src/mame/philips/patgen.cpp

```diff
@@ -0,0 +1,422 @@
+// license:GPL-2.0+
+// copyright-holders: stonedDiscord
+/***************************************************************************
+
+  Philips PM-5xxx test pattern generators
+
+================
+
+V12 80C31 Firmware
+V101 Y Luminance
+V103 Y Luminance
+V105 Y Luminance
+V107 Y Luminance
+V109 Y Luminance
+V201 V Chrominance
+V203 V Chrominance
+V301 U Chrominance
+V303 U Chromninance
```

Comment:
Beyond the obvious typo I'd add a TODO section in header stating what's missing here.